decidim-core/app/views/layouts/decidim/header/_redesigned_menu_breadcrumb_desktop.html.erb

```diff
@@ -7,6 +7,17 @@
   </button>
 </div>
 
+<script>
```

Comment:
Can you move this to a JS file 🙏🏽?

We need to stop this practice of adding JS inline in the HTML. Currently we have a CSP that allows `unsafe-inline` in our   `script-src` policy,  and that's a potential security problem (it allows XSS attacks). The plan is to be able to remove this policy in the future, and for doing so we need to start by not introducing new ones as they'd need to be refactored. 

Also it seems like a big part of the code is the same with the snippet from the other file (`decidim-core/app/views/layouts/decidim/header/_redesigned_menu_breadcrumb_items.html.erb`), so we could probably refactor that to a single function or a similar strategy. What do you think?

Reply:
I was forced to use an inline script, as there's a bug in webkit browsers, but initially my idea was keeping that together (markup + scripts), as I pointed in the comment. Indeed, since there are several concerns about the breadcrumb, I didn't want to put more effort if it's gonna change in close future.

Anyways, I understand your point, I'll do a refactor